protoc-gen-gqlgencfg/main.go

```diff
@@ -25,7 +25,19 @@ func main() {
 		gen.Error(err, "parsing input proto")
 	}
 
-	p := &plugin{Plugin: NewPlugin()}
+	gendir := ""
+	if prefix, ok := Params(gen)["gendir"]; ok {
+		gendir = prefix
+	}
+	idlPrefix := ""
+	if prefix, ok := Params(gen)["idlprefix"]; ok {
```

Comment:
maybe we should have a option with special syntax to replace package prefixes instead of having 2 options? ex: `pkgreplace=old.pkg:new.pkg`